voxblox/include/voxblox/integrator/merge_integration.h

```diff
@@ -139,7 +139,7 @@ void naiveTransformLayer(const Layer<VoxelType>& layer_in,
 
       // find voxel centers location in the output
       const Point voxel_center =
-          T_in_out *
```

Comment:
This was where we thought something was fishy, since the coordinates from `input_block.computeCoordinatesFromLinearIndex` are in the `in` frame, it doesn't make sense to  call `T_in_out * p_in`.

Reply:
Are we taking transforms in voxblox to be active or passive?

Reply:
To the best of my knowledge in voxblox we loosely follow the conventions of maplab/minkindr/aslam_cv, as described in your link below. The transformation uses minkindr and is therefore active. `T_in_out` should correspond to transforming point in the `out` frame to the `in` frame. There are some more examples here [1](https://github.com/ethz-asl/voxblox/blob/master/voxblox/src/integrator/tsdf_integrator.cc#L258), [2](https://github.com/ethz-asl/voxblox/blob/master/voxblox/src/integrator/tsdf_integrator.cc#L326),[3](https://github.com/ethz-asl/voxblox/blob/master/voxblox/src/integrator/tsdf_integrator.cc#L373)
But in terms of notation for points, vectors and such we haven't been following those very closely, e.g. we have points_C and points_W, and below I just added the frame to the variable to make it a bit clearer.

IMO in `transformLayer` and `naiveTransformLayer` the notation was wrong with respect to what the transformation was actually doing, the notation of everything on top of those function, was simply wrong because it was probably switched back and forth until it worked (same for the layer merging unit test i wrote) :)

Reply:
I agree that the notation was wrong with respect to most of voxblox / maplab. I hadn't realized that minkindr was active as the docs for minkindr seem to favor passive representations (https://docs.leggedrobotics.com/kindr/page_rotations.html). 

LGTM